toast-tk-rest-utils/src/main/java/io/toast/tk/core/rest/RestUtils.java

```diff
@@ -273,4 +304,58 @@ public static String downloadRepositoryAsWiki(String apiKey) {
 	public static void unRegisterAgent(String hostName) {
 		// NO-OP
 	}
+
+	private static PoolingHttpClientConnectionManager setHttpConnectionManager() throws NoSuchAlgorithmException, KeyStoreException, KeyManagementException {
+		// This connection manager must be used if more than one thread will
+	       // be using the HttpClient.
+			SSLContextBuilder builder = SSLContexts.custom();
+			builder.loadTrustMaterial(null, new TrustStrategy() {
+			    @Override
+			    public boolean isTrusted(X509Certificate[] chain, String authType)
+			            throws CertificateException {
+			        return true;
+			    }
+			});
+			
+			@SuppressWarnings("deprecation")
+			SSLConnectionSocketFactory sslsf = new SSLConnectionSocketFactory(
+	                builder.build(),SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ALLOW_ALL_HOSTNAME_VERIFIER"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)